### PR TITLE
RenderTarget: Clean up.

### PR DESCRIPTION
**Pull request by linbingquan**
Related issue: #XXXX

**Description**

ES6 default parameters for `RenderTarget`.

### CODE REVIEW

File: src/core/RenderTarget.js [hunk 1]
```diff
@@ -37,20 +37,30 @@ class RenderTarget extends EventDispatcher {
...
 
+		const {
+			generateMipmaps = false,
+			internalFormat = null,
+			minFilter = LinearFilter,
+			depthBuffer = true,
+			stencilBuffer = false,
+			depthTexture = null,
+			samples = 0
+		} = options;
```
Comment (Mugen87) [inline src/core/RenderTarget.js:48]: This is actually destructuring. Not a super fan of this syntax. How about using `Object.assign()` instead?

Comment (linbingquan) [reply src/core/RenderTarget.js:48]: > How about using `Object.assign()` instead?

Thank you for your reply.

I'm sorry, I don't know what you mean, I wonder if you can give the sample code.

Comment (Mugen87) [reply src/core/RenderTarget.js:48]: Check out how `DRACOExporter` defines its default options:

https://github.com/mrdoob/three.js/blob/2c8cfdfc334f5758ee1facd2d95b75e33cd77b18/examples/jsm/exporters/DRACOExporter.js#L22-L32

File: src/core/RenderTarget.js [hunk 2]
```diff
@@ -37,20 +37,30 @@ class RenderTarget extends EventDispatcher {
...
 		this.texture = new Texture( image, options.mapping, options.wrapS, options.wrapT, options.magFilter, options.minFilter, options.format, options.type, options.anisotropy, options.colorSpace );
 		this.texture.isRenderTargetTexture = true;
 
 		this.texture.flipY = false;
-		this.texture.generateMipmaps = options.generateMipmaps !== undefined ? options.generateMipmaps : false;
-		this.texture.internalFormat = options.internalFormat !== undefined ? options.internalFormat : null;
-		this.texture.minFilter = options.minFilter !== undefined ? options.minFilter : LinearFilter;
+		this.texture.generateMipmaps = generateMipmaps;
+		this.texture.internalFormat = internalFormat;
+		this.texture.minFilter = minFilter;
```
Comment (LeviPesin) [inline src/core/RenderTarget.js:56]: Can't we pass these two properties directly to `Texture` constructor?

File: src/core/RenderTarget.js (hunk 1 above)
Comment (LeviPesin) [reply src/core/RenderTarget.js:48]: > Not a super fan of this syntax.

Why not? It seems clearer and simpler to me...

File: src/core/RenderTarget.js (hunk 2 above)
Comment (Mugen87) [reply src/core/RenderTarget.js]: The line with `minFilter` can be removed now.

Comment (linbingquan) [reply src/core/RenderTarget.js]: > The line with `minFilter` can be removed now.

Done.

**Pull request merged**